pkg/wikidata/internal/mappings/wikidata_mapping_structs.go

```diff
@@ -101,3 +97,8 @@ func (wikidata Wikidata) PUIDs() []string {
 	}
 	return puids
 }
+
+// NewWikidata creates new map for adding Wikidata records to.
+func NewWikidata() map[string]Wikidata {
```

Comment:
This fix resolves some problems we were having in the unit tests with state being preserved between runs when we didn't want it to be.